modules/cli/src/main/java/org/apache/ignite/internal/cli/core/repl/completer/filter/NonRepeatableOptionsFilter.java

```diff
@@ -29,8 +30,27 @@ public class NonRepeatableOptionsFilter implements CompleterFilter {
 
     private final CommandSpec topCommandSpec;
 
-    public NonRepeatableOptionsFilter(CommandSpec spec) {
+    private final Map<CommandSpec, List<CompleterFilter>> additionalFilters;
```

Comment:
Additional filters should not be a part of NonRepeatableOptionsFileter. They should be added on a higher level.